soc/xtensa/intel_adsp/Kconfig.defconfig

```diff
@@ -27,4 +27,8 @@ config I2S_CAVS
 	default y
 	depends on I2S
 
+config XTENSA_CCOUNT_HZ
```

Comment:
I'm just curious where this comes from - and why can't this be `CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC`?

Reply:
CCOUNT is a per-CPU special register on Xtensa (technically it's optional, like basically everything[1]) that tracks CPU core cycles (400 MHz on cAVS 2.5).  The same clock is provided to all CPU cores, but the counts aren't synchronized (they start at zero on reset, which happens on power state transitions). 

That's not the same clock as the one provided by the HDA wall clock timer (38.4 MHz), which is a system-wide clock designed for audio synchronization between the various bits of hardware (including the x86 host).

[1] And it would be a good idea to have a framework that predicated this on XCHAL_HAVE_CCOUNT (from core-isa.h) too, though I'm aware of no real world hardware that skips this.